lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -178,3 +178,38 @@ export enum WrapperSKU {
 export const DB_NAME = "msal.db";
 export const DB_VERSION = 1;
 export const DB_TABLE_NAME = `${DB_NAME}.keys`;
+
+export enum CacheLookupPolicy {
+    /*
+     * acquireTokenSilent will first look in the cache before going to the network with the existing refresh token.
+     * The refresh token will be renewed if it is expired.
+     * (existing default behavior)
+     */
+    Default = 0, // 0 is falsy, is equivalent to not passing in a CacheLookupPolicy
+    /*
+     * acquireTokenSilent will only look in the cache, and will not go to the network with the existing refresh token.
+     * The refresh token will not be renewed if it is expired.
+     */
+    AccessToken = 1,
+    /*
+     * acquireTokenSilent will first look in the cache before going to the network with the existing refresh token.
+     * The refresh token will not be renewed if it is expired.
+     */
+    AccessTokenAndRefreshToken = 2,
+    /*
+     * acquireTokenSilent will only go to network with the existing refresh token, and will not look in the cache.
+     * The refresh token will not be renewed if it is expired.
+     */
+    RefreshToken = 3,
+    /*
+     * acquireTokenSilent will only go to network with the existing refresh token, and will not look in the cache.
+     * The refresh token will be renewed if it is expired.
+     * (existing forceRefresh=true behavior)
```

Comment:
Better yet if you can copy the descriptions from the .md file verbatim.
```suggestion
     * equivalent to forceRefresh=true behavior
```